src/apps/external_executor/protobuf/executor_registration.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package ccf;
+
+option optimize_for = LITE_RUNTIME;
```

Comment:
Do we have a rough idea of the performance impact of this setting?

Reply:
None at all. Not particularly attached to it, will likely replace it at least on the KV interface at some point.